docs/models.md

```diff
@@ -0,0 +1,77 @@
+# Risk Prediction, Message Passing, and Clustering
+
+## Overview
+This document describes the code in models. By running `python models/run.py`, you will 
+initiate a message passing algorithm which iterates over the logs output by the mobility simulator. We provide more details
+in the message passing section below. Those logs are described more in `events.md`, but briefly there are four kinds: encounters, symptom_start, contamination,
+and recovery. Our goal is to pass messages between people such that each person can accurately predict their own 
```

Comment:
there are five kinds: encounters, test, symptom_start, contamination, and recovery

Reply:
updated this